rev_news/drafts/edition-46.md

```diff
@@ -96,9 +96,11 @@ This edition covers what happened during the month of November 2018.
 [2.20.0-rc2](https://public-inbox.org/git/xmqq36rhjnts.fsf@gitster-ct.c.googlers.com/),
 [2.20.0-rc1](https://public-inbox.org/git/xmqqmuq25ufc.fsf@gitster-ct.c.googlers.com/),
 [2.19.2](https://public-inbox.org/git/xmqqtvka5ugt.fsf@gitster-ct.c.googlers.com/),
-+ Git for Windows [2.20.1(1)](https://github.com/git-for-windows/git/releases/tag/v2.20.1.windows.1),
-[2.20.0(1)](https://github.com/git-for-windows/git/releases/tag/v2.20.0.windows.1),
-[2.19.2(1)](https://github.com/git-for-windows/git/releases/tag/v2.19.2.windows.1)
++ Git for Windows [2.20.1](https://github.com/git-for-windows/git/releases/tag/v2.20.1.windows.1),
+[2.20.0](https://github.com/git-for-windows/git/releases/tag/v2.20.0.windows.1),
+[2.20.0-rc2](https://github.com/git-for-windows/git/releases/tag/v2.20.0-rc2.windows.1),
+[2.20.0-rc1](https://github.com/git-for-windows/git/releases/tag/v2.20.0-rc1.windows.1),
+[2.20.0-rc0](https://github.com/git-for-windows/git/releases/tag/v2.20.0-rc0.windows.1),
```

Comment:
There should be no comma at the end of the last line related to Git for Windows.

Reply:
I was just imitating what you wrote in the bullet point before: https://github.com/git/git.github.io/pull/330/files/9baa809a2a36f71d12c1f7ca40b00c283f457155#diff-a2cd19c872e78e5e49deec4e466e818fR98

Reply:
Yeah, there were spurious commas in the bullet point before.